libminifi/test/resources/TestC2Metrics.yml

```diff
@@ -31,8 +31,8 @@ Processors:
       run duration nanos: 0
       auto-terminated relationships list:
       Properties:
-          endpoint-list: localhost:8776
-          end-of-message-byte: d
+          Endpoint List: localhost:8776
+          Message Delimiter: \r
           reconnect-interval: 100ms
           connection-attempt-timeout: 2000
```

Comment:
should we update the properties to their new names?

Reply:
sure thing, best not the confuse anyone :+1: https://github.com/apache/nifi-minifi-cpp/pull/1592/commits/ed21b198d8fb54a4f1c483b63cf9f1013ecee5f9